CHANGELOG.md

```diff
@@ -35,6 +41,22 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
   subsequent invocations.
 
 
+* **Added**:
+
+* **Changed**:
+
+* **Deprecated**:
+
+* **Removed**:
```

Comment:
All these headers have a different format than the rest of the file. Also, the headers already exist on `main` now, so you can remove these and put your changes under the existing headers :blush: